app/controllers/register_controller.rb

```diff
@@ -0,0 +1,12 @@
+class RegisterController < Devise::RegistrationsController
+
+  private
+
+  def sign_up_params
```

Comment:
@ksolkowski you have to override the registrations_controller for devise when you add new columns so that it permits them to be saved. is that what you're asking?